profile/src/main/java/org/openedx/profile/presentation/manage_account/compose/ManageAccountView.kt

```diff
@@ -0,0 +1,248 @@
+package org.openedx.profile.presentation.manage_account.compose
```

Comment:
No underscores for package names
https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names
`manage_account` -> `manage`
`manage_account` -> `manageaccount`
or other names